### PR TITLE
Creating a banner on the homepage

### PR DESCRIPTION
**Pull request by ymik0410**
Banner

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <body>
     <%= render 'shared/navbar' %>
     <%= render 'shared/flashes' %>
-    <p class="notice"><%= notice %></p>
+    <p class="notice" hidden><% notice %></p>
```
Comment (casantadu) [inline app/views/layouts/application.html.erb]: We do not need these lines(17; 18)

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,2 +1,4 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class="banner home">
+  <h1>Are you ready for a new cabrio trip?</h1>
```
Comment (casantadu) [inline app/views/pages/home.html.erb:2]: Button to index

**Review by casantadu — COMMENTED** [review]
Hi Yulia, could you please have look at the comments and update accordingly

**Review by casantadu — APPROVED** [review]
Thank you

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (ymik0410) [reply app/views/layouts/application.html.erb]: done

File: app/views/pages/home.html.erb (hunk 2 above)
Comment (ymik0410) [reply app/views/pages/home.html.erb:2]: done

**Pull request merged**